jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/ManagedSession.java

```diff
@@ -51,6 +51,7 @@ public class ManagedSession implements Session
     /**
      * Attribute set if the session is secure
      */
+    @Deprecated
```

Comment:
This static is still used to indicate whether or not a session has been created _before_ a successful authentication.

Reply:
Yeah, that's why only deprecated.    Need to find a better way... or a better place for it.